lib/spack/spack/compilers/gcc.py

```diff
@@ -97,7 +97,10 @@ def default_version(cls, cc):
         if spack.compilers.clang.Clang.default_version(cc) != 'unknown':
             return 'unknown'
 
-        return super(Gcc, cls).default_version(cc)
+        version = super(Gcc, cls).default_version(cc)
+        if version in ['7']:
```

Comment:
`version == '7'`? No need for a list if you only compare a single element.

Reply:
GCC 8 is already in the pipeline; it won't be just GCC 7 doing this.